ScnSoft-Education/LeetCodeTraining.Tests/MergeKSortedListsTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using LeetCodeTraining.MergeKSortedLists;
+
+namespace LeetCodeTraining.Tests
+{
+    public class MergeKSortedListsSolutionTest
+    {
+        [Fact]
+        public void MergeKLists()
+        {
+            // Arrange
+            ListNode list1 = new ListNode(1, new ListNode(4, new ListNode(5)));
```

Comment:
Please use MemberData attribute for tests (please add tests for rest test cases as well), implement constructor to initialize ListNode with int[], also rewrite Equal Method to do assertions directly on ListNode objects.

Reply:
Please implement an constructor as suggested above to initialize ListNode with simple int array and please rewrite TestData  accordingly with applying this constructor.